Sources/MapboxMaps/Gestures/GestureHandlers/PanGestureHandler.swift

```diff
@@ -8,10 +8,16 @@ internal final class PanGestureHandler: GestureHandler {
     // The camera state when the gesture began
     private var initialCameraState: CameraState?
 
+    private var lastChangedDate: Date?
+
+    private let dateProvider: DateProvider
+
```

Comment:
We should add some comments here these properties

Reply:
Updated